src/EventTypeView/EventTypeView.js

```diff
@@ -2,22 +2,110 @@ import React from "react";
 import {connect, PromiseState} from "react-refetch";
 import ConnectionComponent from "../Utils/ConnectionComponent";
 import config from "../config/config";
+import { css }from 'aphrodite';
+import {Col, Row} from "react-grid-system";
+import AppStyles from "../AppStyles";
+import {Card, CardText, CardTitle, IconButton, List, ListItem} from "material-ui";
+import IconSearch from 'material-ui/svg-icons/action/search';
+import Utils from "../Utils/Utils";
 
 class EventTypeView extends ConnectionComponent {
+    static renderEventQueryDetailButton(eventQueryId) {
+        return (
+            <IconButton
+                tooltip={config.descriptions.tooltipToEventQueryInEQMN}
+                href={"/query/" + eventQueryId}>
+                <IconSearch/>
+            </IconButton>
+        );
+    }
+
+    static renderEventQueryList(eventQueries) {
+        return (
+            <List>
+                {
+                    eventQueries.map((eventQuery, key) => {
+                        return (
+                            <ListItem
+                                key={key}
+                                primaryText={eventQuery.query}
+                                disabled={true}
+                                rightIconButton={EventTypeView.renderEventQueryDetailButton(eventQuery.id)}
+                            />
+                        );
+                    })
+                }
+            </List>
+        );
+    }
+
+    static generateEventTypeModel(eventQueries) {
+        const eqmnRepresentation = [];
+        console.log(eventQueries);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=javascript%3AS2228)